Makefile

```diff
@@ -229,10 +229,25 @@ bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metada
 	$(KUSTOMIZE) build config/manifests | $(OSDK) generate bundle $(BUNDLE_GEN_FLAGS)
 	$(OSDK) bundle validate ./bundle
 
+.PHONY: bundle-crd-only
+bundle-crd-only: manifests kustomize operator-sdk ## Generate bundle manifests for CRDs only and metadata, then validate generated files.
+	$(OSDK) generate kustomize manifests -q --interactive=false --input-dir config/manifests-crd  --output-dir config/manifests-crd
```

Comment:
It is very hard to work with such long lines of cryptic commands. Lets have *one* argument per line.

Reply:
Done